am-lib/src/main/java/uk/gov/hmcts/reform/amlib/enums/Permission.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.amlib.enums;
+
+/**
+ * Exposes a set of enum values used to set permissions for Access Management.
+ * Each of the values is a power of two. The reason for that is that in Access Management
+ * there might be multiple permissions that need to be assigned: ie. READ + CREATE.
+ * In order to determine which individual permissions a record has
+ * the binary 'AND' operation is done (the 'isGranted' method).
+ */
+public enum Permission {
+    CREATE(1),
+    READ(2),
+    UPDATE(4),
+    SHARE(8),
```

Comment:
Please also remove this. We will not implement it for next several sprints (maybe never)

Reply:
👍